apps/ledger-live-mobile/.eslintrc.js

```diff
@@ -55,6 +56,7 @@ module.exports = {
     "global-require": 0,
     "no-restricted-globals": 0,
     "no-restricted-syntax": 0,
+    "no-lonely-if": 0,
```

Comment:
it could be valuable to ship these in a separated PR because if some feature gets in unrelated to your "edit ethereum transaction" they would then need to be fixed and you would change things unrelated to it.

basically to make the PR more atomic.